src/pages/PageUsingComponents.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { SplitClient, SplitTreatments } from '@splitsoftware/splitio-react';
+import { feature_1, feature_2, feature_3 } from '../sdkConfig';
+
+/* This example showcasts SplitClient and SplitTreatments components */
+
+const Loading = () => {
+  return <div>Loading SDK...</div>
+}
+
+export default () => {
+  return (
+    <main>
+
+      <SplitTreatments names={[feature_1]} >
+        {({ treatments }) => {
+          return (
+            <div className="App-section">
+              <h4>{`Split: ${feature_1}`}</h4>
+              <p>{`Treatment value: ${treatments[feature_1].treatment}`}</p>
+            </div>
+          )
+        }}
+      </SplitTreatments>
+
+      <SplitClient splitKey="other_user" >
```

Comment:
Add something around this instantiating a different client. No need to explain in detail but mention that this is what you need to do IF you need to evaluate for a different key.